Directory.Build.props

```diff
@@ -1,4 +1,9 @@
 ﻿<Project>
+  <PropertyGroup>
+    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
```

Comment:
Maybe because it's what other repos do? https://github.com/aspnet/Extensions/blob/master/Directory.Build.props. Functionally doesn't make a difference though.

Reply:
RepoRoot needs to be set before Arcade's Sdk.props file is imported.